### PR TITLE
Added tests for book and fix flake8

### PR DESCRIPTION
**Pull request by volodymyr-komarnyckyi**

### CODE REVIEW

File: book/tests.py [hunk 1]
```diff
@@ -1,3 +1,157 @@
...
+        self.assertIn("image", res.data)
+        self.assertTrue(os.path.exists(self.book.image.path))
+
+    def test_upload_image_bad_request(self):
+        url = image_upload_url(self.book.id)
+        res = self.client.post(url, {"image": "not image"}, format="multipart")
+        self.assertEqual(res.status_code, status.HTTP_400_BAD_REQUEST)
+
+
+class UnauthenticatedBookTests(TestCase):
```
Comment (anonymousprojectuser) [inline book/tests.py:68]: Add test, that unauthenticated user cannot create new book

File: book/tests.py [hunk 2]
```diff
@@ -1,3 +1,157 @@
...
+    def setUp(self) -> None:
+        self.client = APIClient()
+
+    def test_auth_required(self):
+        res = self.client.get(BOOK_URL)
+
+        self.assertEqual(res.status_code, status.HTTP_200_OK)
```
Comment (anonymousprojectuser) [inline book/tests.py:75]: Add test, that unauthenticated user cannot create new book

File: book/tests.py [hunk 3]
```diff
@@ -1,3 +1,157 @@
...
+
+        url = detail_url(book.id)
+        res = self.client.get(url)
+
+        serializer = BookSerializer(book)
+
+        self.assertEqual(res.status_code, status.HTTP_200_OK)
+        self.assertEqual(res.data, serializer.data)
+
+    def test_create_movie_forbidden(self):
```
Comment (anonymousprojectuser) [inline book/tests.py]: rename the name of test:)

File: book/tests.py [hunk 4]
```diff
@@ -1,3 +1,157 @@
...
+        self.user = get_user_model().objects.create_user(
+            email="admin@myproject.com",
+            telegram_nick="@Volodymyr2856",
+            password="password",
+            is_staff=True
+        )
+
+        self.client.force_authenticate(self.user)
+
+    def test_create_movie(self):
```
Comment (anonymousprojectuser) [inline book/tests.py]: rename here also:)

**Review by anonymousprojectuser — APPROVED** [review]

**Review by anonymousprojectuser — APPROVED** [review]

**Review by kovaliskoveronika — APPROVED** [review]

**Pull request merged**